requirements-dev.txt

```diff
@@ -2,4 +2,8 @@
 ruff
 pre-commit
 pip-tools
-pip-compile-cross-platform
\ No newline at end of file
+pip-compile-cross-platform
+pytest
+pytest-cov
+pytest-asyncio
+psycopg2-binary
```

Comment:
Don't know why this is required for testing only not for the application itself. 🤔
It was erroring for `psycopg2` but I couldn't install it as it recommended installing `psycopg2-bianry`